pootle/apps/pootle_language/views.py

```diff
@@ -68,13 +83,9 @@ def language(self):
             'code': self.object.code,
             'name': tr_lang(self.object.fullname)}
 
-    def get(self, *args, **kwargs):
-        response = super(LanguageBrowseView, self).get(*args, **kwargs)
-        response.set_cookie('pootle-language', self.object.code)
-        return response
-
 
 class LanguageTranslateView(LanguageMixin, PootleTranslateView):
+    page_path = "pootle-language-translate"
```

Comment:
One thing I'd like to hear from others is if in terms of naming `page_path` feels appropriate or not. As we see these class attributes contain the URL pattern name that is used by `reverse()` (via `redirect()` here) to generate URLs, therefore it'd maybe be more understandable to name this class attribute as `url_pattern_name` or similar. Thoughts?